src/language-select/components/LanguageSelect/LanguageSelectListItem.tsx

```diff
@@ -161,7 +162,19 @@ export const LanguageSelectListItem: React.FC<Props> = ({status, language}) => {
               },
             }
           : {}),
-      })
+      }
+
+      const translatedDocument = config.onTranslationCreate
```

Comment:
I could imagine that in the future we might want to have more of an emitter kind of thing, where there could be multiple functions called. But that's for later.